client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -42,7 +42,7 @@
             <dp-radio
               name="r_role"
               value="0"
-              data-cy="simplifiedNewStatementForm:citizenButton"
+              data-cy="roleInput:role.citizen"
```

Comment:
```suggestion
              data-cy="roleInput:citizen"
```

Reply:
But now we have different hooks names between projects!
Tester needs similar names in all projects!
in EWM: `data-cy="roleInput:citizen"`
in BLP or another projects: `data-cy="roleInput:role.citizen"`, see `DpAutofillSubmitterData.vue`
https://demoseurope.youtrack.cloud/issue/DPLAN-11846

Reply:
okay, moment, Commit is coming!

Reply:
https://github.com/demos-europe/demosplan-core/pull/3280/commits/413b4933baebdf6c77041195f6bef07d357ede0e